<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Template1</class>
 <widget class="QDialog" name="Template1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="btn01">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>223</y>
     <width>76</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn02">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>223</y>
     <width>76</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn03">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>223</y>
     <width>76</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn04">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>223</y>
     <width>76</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn05">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>223</y>
     <width>76</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn06">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>223</y>
     <width>76</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="img01">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>84</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="img02">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>70</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgtitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Saab</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text01">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>84</y>
     <width>310</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text02">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>112</y>
     <width>310</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text03">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>146</y>
     <width>310</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text04">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>194</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progbar">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>197</y>
     <width>270</width>
     <height>14</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="sbar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>250</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
